internal/handlers/handlers_test.go

```diff
@@ -297,3 +319,147 @@ func TestPostBatchDB_EmptyRequest(t *testing.T) {
 		t.Errorf("ожидался статус %d, но получен %d", http.StatusNotFound, w.Code)
 	}
 }
+
+func TestHandlers_GetUsersURLs(t *testing.T) {
+	tests := []struct {
+		name         string
+		expectedCode int
+		userURLs     []*models.UserURLs
+		expectedErr  error
+		ctx          bool
+	}{
+		{
+			name: "successful_get",
+			userURLs: []*models.UserURLs{
+				{
+					ShortURL:    "test",
+					OriginalURL: "original_test",
+				},
+			},
+			expectedCode: 200,
```

Comment:
 Коды можно заменить на константы.
```suggestion
			expectedCode: http.StatusOK,
```